src/OnboardingSPA/components/LivePreviewSkeleton/index.js

```diff
@@ -0,0 +1,38 @@
+import { useSelect } from '@wordpress/data';
```

Comment:
Similar to the comment in the state PR this component needs to go inside `LivePreview` components and then imported from there similar to `BlockPreview` and our `SelectableCards`